### ISSUE TITLE
commentary study

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/PWK#39]
https://github.com/sanskrit-lexicon/PWK/tree/master/pw_ls/pw_dhaval/abbrvwork/commentary

This subrepository tries to analyse the commentary references in pw.txt

As of now
```
A. grep ' ‹zu› ' ../../../../../Cologne_localcopy/pw/orig/pw.txt > commentaryrefs.txt

This command fetches all the ' ‹zu› ' occurrences from pw.txt and stores to commentaryrefs.txt

B. Run commentarystudy.py 

This takes commentaryrefs.txt as input and gives the following output. 
B1. commleft.txt - references where the pattern matches '•Comm. ‹zu› ¯([A-Za-z0-9.,]*)'
```

**Comment by gasyoun** [comment]
I'll not dive into this ocean for now, seems to deep. There should be a list of "service" words, not only `zu`, but `an`, `fgg.`. But as it is even above general abbreviation research, I must understand that alone I can't manage all of the already open issues. I would first finish the list of 12 lacking references in PW.

**Comment by funderburkjim** [comment]
@drdhaval2785  Skipping this issue for now.

**Comment by drdhaval2785** [comment]
Yes. I am trying to refine the python code to get some more easily workable and manageable items.
Please leave it as it is now.

**Comment by gasyoun** [comment]
**Question type:** editorial-policy / citation

**Evidence found:**
The issue is about commentary references in pw.txt using the pattern `Comm. ‹zu›` (commentary on). Checked csl-orig/v02/pw/pw.txt: 2710 occurrences of `Comm.`. The linked file [commleft.txt](https://github.com/sanskrit-lexicon/PWK/blob/master/pw_ls/pw_dhaval/abbrvwork/commentary/commleft.txt) extracts references matching `•Comm. ‹zu› ¯([A-Za-z0-9.,]*)`. Thread conclusion (2016-01): @gasyoun and @funderburkjim agreed to skip/defer this work; @drdhaval2785 was refining the Python code to get more workable output.

**Assessment:**
Evidence inconclusive — needs human editorial decision: the commentary study was explicitly deferred in 2016. The core question (how to handle `Comm. ‹zu›` references where the cited text is a numeral or unusual string) was noted but not resolved. The commleft.txt analysis file exists in the repo but no follow-up action was taken.

**Suggested next step:** If resuming this work, check whether commleft.txt is still current with the pw.txt state in csl-orig, then triage the numeral-headed entries (e.g. entries starting with a digit) to determine if they are errors or valid commentary references on numbered texts.
